src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/XamlReaderConstants.cs

```diff
@@ -27,7 +30,7 @@
 using Microsoft.Win32;
 using System.Runtime.InteropServices;
 using MS.Internal;
-
+*/
```

Comment:
nit: can we not remove these usings instead of commenting them?

Reply:
Hmm. Missed this one apparently. I'll fix it.